tests/xfpga/test_sysfs_c.cpp

```diff
@@ -292,9 +264,51 @@ INSTANTIATE_TEST_SUITE_P(sysfsinit_c, sysfsinit_c_p,
                                                                         "dfl-d5005",
                                                                         "dfl-n3000",
                                                                         "dfl-n6000-sku0",
-                                                                        "dfl-n6000-sku1"
+                                                                        "dfl-n6000-sku1",
+                                                                        "dfl-c6100"
                                                                       })));
 
+class sysfsinit_bbsid_c_p :public  sysfsinit_c_p
```

Comment:
Just curious, why did this need to be moved?

Reply:
C6100 doesn't support PR, get_interface_id is not valid test cases
get_interface_id test case works for N6000, D5005 pcie cards.